browser/base/content/newtab/newTab.js

```diff
@@ -18,8 +17,12 @@
     "resource://gre/modules/PrivateBrowsingUtils.jsm");
   XPCOMUtils.defineLazyModuleGetter(imports, "Services",
     "resource://gre/modules/Services.jsm");
+  XPCOMUtils.defineLazyModuleGetter(imports, "RemoteNewTabLocation",
+    "resource:///modules/RemoteNewTabLocation.jsm");
 
   let iframe;
+  const TRUSTED_ORIGIN = imports.RemoteNewTabLocation.origin;
```

Comment:
I don't think we need these two constants, just `imports.RemoteNewTabLocation.origin` where needed